File: 2928. Distribute Candies Among Children I.ts
function distributeCandies(n: number, limit: number): number {
    let count = 0;
    let result = []
    for(let i =0;i<=limit;i++){
        for(let j=0;j<=limit;j++){
            for(let k=0;k<=limit;k++){
                let sum = [i,j,k].reduce((a,b) => a+b)
                if(sum==n){
                    count +=1
                    result.push([i,j,k])
                }
            }
        }
    }
    return count
};

File: 2928. Distribute Candies Among Children I.dart
class Solution {
  int distributeCandies(int n, int limit) {
    var count = 0;
    for(var i =0;i<=limit;i++){
        for(var j=0;j<=limit;j++){
            for(var k=0;k<=limit;k++){
                var sum = [i,j,k].reduce((a,b) => a+b);
                if(sum==n){
                    count +=1;
                }
            }
        }
    }
    return count;
  }
}
